pkg/kube/cluster-init.sh

```diff
@@ -74,13 +76,14 @@ apply_multus_cni() {
         get_default_intf_IP_prefix
         kubectl create namespace eve-kube-app
         logmsg "Apply Multus, Node-IP: $NODE_IP"
-        while ! kubectl apply -f /tmp/multus-daemonset.yaml; do
-                sleep 1
-        done
+        if ! kubectl apply -f /tmp/multus-daemonset.yaml; then
```

Comment:
I intentionally changed this to `while` because I didn't like the `sleep 10` that we would add after `apply_multus_cni`
See below:
```
                apply_multus_cni
                sleep 10
                continue
```

Reply:
CC @naiming-zededa (who originally wrote this part of cluster-init.sh)

Reply:
Is there a reason that the `kubectl apply` would fail? I could see how the actual multus readiness might take some time due to kubernetes' controller loop nature of "apply and then return and then check later if it is complete"  (hence the `sleep 10`, I believe?), but you shouldn't need to apply it multiple times.

Is this a question of "apply until the apply itself succeeds", in which case the `while` statement is a better fit? Or is this a question of "apply which should succeed right away, but wait for the deployment of containers to complete", in which case the `if` statement is a better fit?

If the latter, maybe `kubectl rollout status` along with a `--timeout`?

Reply:
The sleep can be decreased

Reply:
This while loop would in some cases stuck forever, since this k3s may have already crashed due to some reasons, we have seen this often. we may not need to sleep 10 seconds, but we need to 'continue' from the main loop to relaunch the k3s if needed.

Reply:
I'll trim the sleeps a bit, it was a balance of checking often enough to keep forward progress but not so often that we're just spinning cpu or kube api time with no updates.

Reply:
@deitch I see your note about rollout status, that may be a clean method for us to check multus status.  for example on my system:
```
ecde95a4-0f1b-4f6a-a470-035fd6a79057:/# kubectl -n kube-system rollout status daemonset/kube-multus-ds-amd64
daemon set "kube-multus-ds-amd64" successfully rolled out
```

Reply:
@milan-zededa I trimmed the sleep in latest commit.  
@deitch I'll keep in mind using kubectl rollout status in a later refactoring.

Reply:
OK, thanks